src/test/chainparamsbase_tests.cpp

```diff
@@ -15,7 +15,7 @@ extern void noui_connect();
 
 struct ChainParamsBaseTestingSetup {
```

Comment:
You should also rename to FederationParamsBaseTestingSetup. filename and following code too.

Reply:
yes! will correct it.